### PR TITLE
feat(client & devtools-kit): add route-meta for pages

### PR DESCRIPTION
**Pull request by wzc520pyfm**
hi~! I see #330 , this pr add route meta for pages.

Preview:

https://github.com/user-attachments/assets/c39e3dd7-77b7-4303-8fbf-eee80f29fbde

### CODE REVIEW

File: pnpm-lock.yaml
Comment (wzc520pyfm) [inline pnpm-lock.yaml:1]: I'm not sure if I should commit the lock file.

**Comment by alexzhang1030** [comment]
the bundle size of `json-editor-vue` is above 500kb, this is huge... Since this is not a feature for high-frequency use, and most of the time is read instead of edited, how about just using `Shiki` for render meta info? We have a `CodeBlock` component on the `@vue/devtools-ui` package.

**Comment by wzc520pyfm** [comment]
> the bundle size of `json-editor-vue` is above 500kb, this is huge... Since this is not a feature for high-frequency use, and most of the time is read instead of edited, how about just using `Shiki` for render meta info? We have a `CodeBlock` component on the `@vue/devtools-ui` package.

Sorry for delay. I have replaced `json-editor-vue` with `CodeBlock`.

**Review by alexzhang1030 — APPROVED** [review]
LGTM!

A screen record:

https://github.com/user-attachments/assets/20ac178d-88d5-4f30-8190-d8b6fc62528c

**Comment by webfansplz** [comment]
Because the route meta info does not exist in every project, I think it is better UX to hide this field if all the routes of the user project do not have this field.

**Review by webfansplz — APPROVED** [review]

**Pull request merged**